src/Symfony/Bridge/PhpUnit/Tests/ClockMockTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Tests;
+
+use Symfony\Bridge\PhpUnit\ClockMock;
+
+class ClockMockTest extends \PHPUnit_Framework_TestCase
+{
+    public function testTimeNowNull()
+    {
+        $this->assertTrue(is_int(ClockMock::time()));
+    }
+
+    public function testSleepNowNull()
+    {
+        $this->assertEquals(0, ClockMock::sleep(0));
```

Comment:
Done in tests below. I don't want to change $now in above tests, because I want to cover
```
        if (null === self::$now) {
        }
```
cases.